### ISSUE TITLE
Swift 2.3 or Swift 3 syntax update.

### ISSUE THREAD

**Issue by 128keaton** [PhamBaTho/BTNavigationDropdownMenu#70]
Hey, will this be updated anytime soon?

**Comment by PhamBaTho** [comment]
@128keaton Yeah, absolutely. Syntax will be updated as soon as new Swift version comes out. Just like syntax "#selector" of Swift 2.2 was updated in this library.

**Comment by tulioht** [comment]
I Need in Swift 3, XCode 8 GM is out

**Comment by CliveTeow** [comment]
Is this completed?

**Comment by 128keaton** [comment]
If you are so impatient, you can update it yourself :). Its surprisingly easy.

**Comment by PhamBaTho** [comment]
@tulioht @CliveTeow @128keaton I fixed this issue in [Swift-3.0 branch](https://github.com/PhamBaTho/BTNavigationDropdownMenu/tree/swift-3.0). 

From now, you can use these lines of code in your Podfile:

```
pod 'BTNavigationDropdownMenu', :git => 'https://github.com/PhamBaTho/BTNavigationDropdownMenu.git', :branch => 'swift-3.0'
```
